data_structures/binary_tree/binary_search_tree.py

```diff
@@ -141,6 +141,18 @@ def traversal_tree(self, traversal_function=None):
         else:
             return traversal_function(self.root)
 
+    def inorder(self, arr, Node):
```

Comment:
@SiddhantBobde Kindly add docstrings and the type of parameters passed and their return types, you can refer here https://www.geeksforgeeks.org/python-docstrings/#:~:text=Python%20documentation%20strings%20(or%20docstrings,a%20specific%20segment%20of%20code.

eg:-

def sum(a,b):
     """ The sum func does the summation of a and b.
    Args:-
       a -> int
       b -> int
    Returns : - int
    """"
     return a+b

or 

def sum(a : int,b : int) -> int: 
     """ The sum func does the summation of a and b."""
     return a+b

Reply:
ok, I will add docstrings and type of parameters.

Reply:
doctests are different that docstrings.
* https://docs.python.org/3/library/doctest.html
type hints are different than comments about the type of parameters.
* https://docs.python.org/3/library/typing.html

Reply:
ok, I will correct it.